Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "coldsnap"
-version = "0.3.3"
```

Comment:
We should probably leave the version bump for a separate PR once we have a changelog and we're ready to release a new crate.